content/md/imports-exports/monitor-jobs.md

```diff
@@ -9,50 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
+Please note it is possible to use the `View all jobs in Process Tracker` permission in the `Roles` page to access all the jobs launched by all the users of your PIM. To find out more, check our article on [permissions](manage-the-interface-and-actions-accesses.html#rights-on-system-pages) in our help center.
 :::
 
-## Imports & exports reports
+# What can you find in a job detail?
+
+A job detail is a report of a specific job.
+In each job detail, you'll find information about the status of your job, which may vary depending on their status: `Started`, `Completed`, `Failed` or `Stopped` with a specific color code. You will see:
+* The number of steps
+* A time estimation above each step
+* A table with the details of all the actions performed under each step
 
-You can also have more details when you executed an import or an export. To review their report:
-1.  Go to `Imports`
-2.  Select in the list the profile job for which you want to view the report
-2.  You can clearly see the status of your import: `Completed`, `Started` or `Failed` thanks to a color code
-1.  Click on your import/export
-1.  The page of the import/export job detail should appear
+To access a specific job detail:
+1.  Go to `Activity`/`Process Tracker`
+1.  Search for the job you would like to know more about
+1.  Click on the job
+![Job detail example](../img/Jobs_job-detail.png)
 
-![image](../img/Exports_Calculation5.png)
+# How to monitor your jobs?
+## Import & export jobs
 
-The details of the executing import/export job are shown on the page. The details are related to each executed import/export profile.
+You can get more details when you execute an import or an export job. To access the job detail:
+1.  Go to `Imports` or `Exports`
+1.  In the list, select the profile job you need
+1.  Click on your import/export job
+1.  The import/export job detail appear
+![Jobs export/import profiles](../img/Jobs_export-import-profiles.png)
 
-## Mass actions execution reports
+The details of the executed import/export job are displayed and related to individual import/export profiles.
 
-You can also have more details when you executed a mass action. To review their report:
-1.  Go to `Activity`/`Process Tracker` or click on the notification of the completed mass action
-1.  The page of the report is then displayed: the details of the executing bulk action are shown on the page
+## Mass actions jobs
 
-![image](../img/Dashboard_Calculation6.png)
+You can have more details about executing a mass action. To check the report:
+1.  Go to `Activity`/`Process Tracker`
+1.  Search for the job you would like to know more about
+1.  Click on the job to get more details
+![Jobs mass execution details](../img/Jobs_mass-actions.png)
```

Comment:
The image has weird grey areas at the top and bottom. Maybe coming from the Figma filter?